### PR TITLE
Update Installation Guide to better reflect the current state of the project

### PR DESCRIPTION
**Pull request by fidencio**
This PR aims to update the installation information for the distros claimed to be supported.

The main changes here are:

 Drop community packages in favour of kata-deploy;
 CentOS: use packages from the virtualization special interest group;
 Fedora: use the packages provided by the distro;
 openSUSE: use the packages provided by the distro;
 RHEL: drop the information;
 SLES: use the packages provided by the distro;

### CODE REVIEW

**Review by jodh-intel — APPROVED** [review]
Thanks @fidencio.

File: tools/packaging/README.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 * [Introduction](#introduction)
 * [Build using OBS](#build-using-obs)
 * [Build in a container](#build-in-a-container)
-* [Build a snap package](#build-a-snap-package)
```
Comment (amshinde) [inline tools/packaging/README.md:6]: @fidencio I am not sure about this. I understand we decided to move away from OBS packages, but we still would want to keep snap packaging for debian/ubuntu.

cc @egernst @bergwolf @jodh-intel @devimc

Comment (fidencio) [reply tools/packaging/README.md:6]: I've updated the series dropping the patch that would drop the snap support, my bad for getting to eager and deleting that as well. :-)

**Comment by fidencio** [comment]
@amshinde, may I bother you for a review?

File: docs/install/opensuse-installation-guide.md [hunk 2]
```diff
@@ -3,19 +3,7 @@
...
-   $ ARCH=$(arch)
-   $ BRANCH="${BRANCH:-master}"
-   $ REPO_ALIAS="kata-${BRANCH}"
-   $ PUBKEY="/tmp/rpm-signkey.pub"
-   $ curl -SsL -o "$PUBKEY" "https://raw.githubusercontent.com/kata-containers/tests/master/data/rpm-signkey.pub"
-   $ sudo -E rpm --import "$PUBKEY"
-   $ zypper lr "$REPO_ALIAS" && sudo -E zypper -n removerepo "$REPO_ALIAS"
-   $ sudo -E zypper addrepo --refresh "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/${DISTRO_REPO}/" "$REPO_ALIAS"
-   $ sudo -E zypper -n install kata-runtime
+   $ sudo -E zypper -n install katacontainers
```
Comment (bergwolf) [inline docs/install/opensuse-installation-guide.md:6]: Are we calling it `kata-runtime` on Centos/RHEL and `katacontainers` on openSUSE/SLES?

File: docs/install/README.md [hunk 3]
```diff
@@ -35,27 +35,16 @@ to see if your system is capable of running Kata Containers.
...
+|Distribution (link to packages)                          | Versions                                                                       | Contacts                                              |
+|---------------------------------------------------------|--------------------------------------------------------------------------------|-------------------------------------------------------|
+|[CentOS](centos-installation-guide.md)                   | 8                                                                              |                                                       |
+|[Fedora](fedora-installation-guide.md)                   | 32, Rawhide                                                                    |                                                       |
+|[SUSE Linux Enterprise (SLE)](sle-installation-guide.md) | SLE 15 SP1, 15 SP2                                                             |                                                       |
+|[openSUSE](opensuse-installation-guide.md)               | [Leap 15.1](opensuse-leap-15.1-installation-guide.md)<br>Leap 15.2, Tumbleweed |                                                       |
```
Comment (bergwolf) [inline docs/install/README.md:47]: Where do we build these packages now? Are they built by distribution package maintainers?

**Review by amshinde — APPROVED** [review]

File: docs/install/opensuse-installation-guide.md (hunk 2 above)
Comment (amshinde) [reply docs/install/opensuse-installation-guide.md:6]: @bergwolf Looks like it. I think its worthwile to have this consistent in the long run.

**Comment by amshinde** [comment]
@fidencio lgtm overall, I havent gone back and looked at how we land on the individual installation pages, but want to make sure we still mention `kata-deploy` method is available for installation as well to try out the latest release.

File: docs/install/opensuse-installation-guide.md (hunk 2 above)
Comment (fidencio) [reply docs/install/opensuse-installation-guide.md:6]: On Fedora / CentOS / RHEL we made the decision of having one package for each component (kata-runtime, kata-shim, kata-osbuilder, kata-agent, ...). Looking at this decision now I clearly see how wrong we were. :-)

Hopefully we'll get things right for 2.0, and be consistent with the katacontainers name.
@etrunko, let's remember this for the future!

File: docs/install/README.md (hunk 3 above)
Comment (fidencio) [reply docs/install/README.md:47]: We don't build those anymore. They're built by the distro packages maintainer.

**Review by bergwolf — APPROVED** [review]

**Pull request merged**